src/Athena/Views/Home/Add_Classes.cshtml

```diff
@@ -2,29 +2,90 @@
+                            <p>I am a very simple card. I am good at containing small bits of information. I am convenient because I require little markup to use effectively.</p>
+                        </div>
+                    </div>
+                </li>
+            </ul>
+        </div>
+        <div class="row">
+            <h1 class="btn col s6 offset-s3">All Courses</h1>
+        </div>
+        <div>
 
-<div id="first">
-    <div class="row">
-        <h1 id="specific" class="btn col s6 offset-s3">Degree Specific Courses</h1>
-    </div>
-    <div>
-        <p>First Section Content</p>
+            <ul class="classUnordered">
+                <li>
+                    <div class="card courseCard">
+                        <span class="card-title courseCardTitle">Math</span>
+                        <a class="btn-floating halfway-fab waves-effect waves-light blue darken-4" onclick="changesymbol(this, this.id, 'Math', '12:30', '14:30', [1, 3, 5])" id="1520"><i class="material-icons">add</i></a>
+                        <div class="card-content courseCardContent">
+                            <p>Math</p><p>1520</p><p>12:30-2:30</p><p>Mon,Wed,Fri</p>
+                        </div>
+                    </div>
+                </li>
+                <li>
+                    <div class="card courseCard">
+                        <span class="card-title courseCardTitle">Physics</span>
+                        <a class="btn-floating halfway-fab waves-effect waves-light blue darken-4" onclick="changesymbol(this, this.id, 'Physics', '8:00', '10:00', [1, 3, 5])" id="1234"><i class="material-icons">add</i></a>
+                        <div class="card-content courseCardContent">
+                            <p>Physics</p><p>1234</p><p>8:00-10:00</p><p>Mon,Wed,Fri</p>
+                        </div>
+                    </div>
+                </li>
+                <li>
+                    <div class="card courseCard">
+                        <span class="card-title courseCardTitle">Reading</span>
+                        <a class="btn-floating halfway-fab waves-effect waves-light blue darken-4" onclick="changesymbol(this, this.id, 'Reading', '16:30', '18:30', [2, 4])" id="3004"><i class="material-icons">add</i></a>
+                        <div class="card-content courseCardContent">
+                            <p>I am a very simple card. I am good at containing small bits of information. I am convenient because I require little markup to use effectively.</p>
+                        </div>
+                    </div>
+                </li>
+            </ul>
+        </div>
+        <div class="row">
+            <h1 class="btn col s6 offset-s3">Completed Courses</h1>
+        </div>
+        <div>
+            <p>Second Section Content</p>
+        </div>
     </div>
+    <br />
     <div class="row">
-        <h1 class="btn col s6 offset-s3">All Courses</h1>
+        <a href="User_Page" class="btn col s6 offset-s3">Finish</a>
```

Comment:
Use `@Url.Action(...)` to link to other pages.

Reply:
changed